Teradata/covalent
# Feature/stepper pattern

## Description
Created a design pattern for vertical stepper

### What's included?
- Stackblitz demo
- New Pattern in Docs

#### Test Steps

- [] Open http://localhost:4200/#/design-patterns/steppers
- [] Fill out form
- [] Verify link to code editor works (top right of Example card)
- [] Verify guidance is accurate

#### General Tests for Every PR
- [ ] `npm run build:release` still works.

##### Screenshots or link to StackBlitz/Plunker
![stepperpattern](https://user-images.githubusercontent.com/17860952/34133124-eae74b26-e407-11e7-9985-76eb7f489724.gif)
